### ISSUE TITLE
DM2E 1.1 rev15: edm:currentLocation and dm2e:shelfmarkLocation as edm:Place

### ISSUE THREAD

**Issue by ksdm2e** [DM2E/dm2e-mappings#57]
In rev.1.5 the range of of edm:currentlocation and dm2e:shelfmarkLocation was changed to edm:Place. According to p. 67, edm:Place is "extent in space, in particular on the surface of the earth, in the pure sense of physics".

Locating a certain issue of a work is not a matter of geo referencing, but the correlation to a holding institution (edm:Agent) and an identifier (shelfmark, german: "Signatur"). 

``` xml
<edm:currentLocation rdf:resource="http://data.dm2e.eu/data/agent/bbaw/dta/Staatsbibliothek%20zu%20Berlin%20%E2%80%93%20Preu%C3%9Fischer%20Kulturbesitz"/>
<dm2e:shelfmarkLocation>SBB-PK, 19 ZZ 3300</dm2e:shelfmarkLocation>
```

The range should be changed to edm:Agent and rdf:Literal.

**Comment by d0rg0ld** [comment]
I agree with @ksdm2e , +1

**Comment by edroege** [comment]
I general, it would be good to have either resources or literals as ranges. If not, we have problems with the owl representation. Can we just have Literal and I will make it as a subproperty from something else than currentLocation

**Comment by ksdm2e** [comment]
Maybe my last sentence was missleading. 

I meant range of 
- edm:currentLocation: edm:Agent, edm:Place, edm:WebResource(?)
- dm2e:shelfmarkLocation: rdf:Literal

That would be sufficient for our usage and interpretation.

**Comment by edroege** [comment]
The definition of edm:currentLocation is "Current location of physical CHO, possibly a library building." (DM2E) and "The geographic location and/or name of the repository, building, site, or other entity whose boundaries presently include the resource." I would not read it as if agents should be allowed here and in my opinion, we would mix the semantics a lot up by adding agents and web resources. 

We have the property dm2e:owner which could be used to indicate which agent holds the CHO. If this property does not fit because the owner is not always the person who holds the object (if you have such cases, please provide an example) then we could add a new property like dm2e:holdingInstitution. 

I agree that shelfmarkLocation needs a Literal. Then it cannot be a subproperty of edm:currentLocation anymore. I am not yet sure where we could place it in the model (neccessary for backwards compatibility with EDM) but will think of it and propose something here.

**Comment by knepper** [comment]
As far as I understand it we will need three properties if we want to cover everything:
- edm:currentLocation with definition of europeana: "The geographic location whose boundaries presently include the CHO. If the name of a repository, building, site, or other entity is used then it should include an indication of its geographic location."

``` xml
<edm:currentLocation rdf:resource=“http://sws.geonames.org/2950159”/> (Identifier for Berlin)
```
- Owner and holder: In our case the holder and owner are different. Holder of the manuscripts is the university library (owned by the state Hassia) but the owner of the manuscripts is the city of Frankfurt am Main.

**Comment by ksdm2e** [comment]
@edroege I also think "possibly a library building" is creepy, because it mixes a geo location and the conceptual location in the the library system.

Also, "The geographic location and/or name of the repository" is ambivalent. Because in my case I could argue, that "the name of the repository" is mere an agent than a place.

However, _holdingInstitution_ came also to my mind once before, so I would give a 1+ for that.

**Comment by edroege** [comment]
Thank you for your input! To summarise it: We will add a new property dm2e:holdingInstitution (domain CHO, range foaf:organization), change the range from dm2e:shelfmarkLocation to literal and make the dm2e:scopeNote of edm:currentLocation clearer ("Current geographic location of physical CHO.") in model rev. 1.6.

**Comment by kba** [comment]
Fixed in 1.1_Rev1.6 of the validator, as of build 'Wed Mar  5 11:10:49 CET 2014'
